### PR TITLE
Update functionality and clean up code from the previous pull request comments

### PR DESCRIPTION
**Pull request by kaja-osojnik**
Fixes more comments from https://github.com/oasisprotocol/playground/pull/29

- Change the naming of Language and Tags components for easier distinction
- Change Figtree font import
- Separate Language and Tags in filters and project items
- Add copyright information at the bottom
- Remove unused dependencies

Screenshots
Before:
Screenshot 2024-01-15 at 15 43 49

After:
Screenshot 2024-01-15 at 15 45 17

### CODE REVIEW

**Review by matevz — DISMISSED** [review]
I rebased locally to the `main` branch and it seems the "Authenticator demo" is missing. Perhaps there are some silent parsing errors when building and it's not included? Or maybe the markdown syntax bothers the parser and you need to merge in the markdown support first?

The screenshot from the main branch:

image

Your branch:

image

File: src/components/ProjectList.tsx [hunk 1]
```diff
@@ -145,17 +139,18 @@ const ProjectList: React.FC = () => {
             return false;
           });
 
+          const paratimeMatch: boolean =
+          selectedParatimes.length > 0 &&
+          (selectedParatimes.length === 0 ||
```
Comment (lukaw3d) [inline src/components/ProjectList.tsx:144]: This doesn't make sense

File: src/components/Filters.tsx [hunk 2]
```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import { Box, Button, Checkbox, FormControlLabel, Grid, Switch, Typography } from '@mui/material';
-import Tags from './Tags';
+import TagFilterList from './TagFilterList';
+import LanguageFilterList from './LanguageFilterList';
```
Comment (lukaw3d) [inline src/components/Filters.tsx:4]: I would name them TagsSelector and LanguagesSelector

**Comment by lukaw3d** [comment]
@matevz when I rebase I see Authenticator

**Review by lukaw3d — APPROVED** [review]

**Comment by matevz** [comment]
> @matevz when I rebase I see Authenticator

Yes, this was fixed in https://github.com/oasisprotocol/playground/pull/36

**Pull request merged**